### DISCUSSION TITLE
Set featured image on post but not have it show up in the post itself

### DISCUSSION THREAD

**Discussion by jjets718** [GetPublii/Publii#1660] [Help]
Hi there! I want to give each of my posts a featured image. I want the featured image to show up in the post previews on the homepage but not in the posts themselves. Is there a way to do that? I don't see a way to do that in the settings so I'm curious if anyone has found a workaround. Thank you in advance!

**Comment by bjazmoore** [comment]
You could edit the theme.  That seems to be the only option currently.  Consider making this a feature request.

**Comment by candidexmedia** [comment]
Which theme are you using? I've been able to do this with Mercury by [customizing the theme's files](https://forum.getpublii.com/topic/how-to-remove-featured-picture-from-posts/#post-8596).

**Comment by michaelfluegge** [comment]
https://forum.getpublii.com/topic/how-to-remove-featured-picture-from-posts/

restart the App after making the changes

**Reply by kukkurovaca** [reply]
This is super useful, thank you!

**Comment by s3nnet** [comment]
If you just want a different image for the post itself, you could use the new "custom featured image" feature:
https://getpublii.com/blog/release-046.html

**Reply by michaelfluegge** [reply]
In case the theme supports the new feature

**Reply by s3nnet** [reply]
That is true. I am using TechNews with Publiii 46, which does not seem to support it, which is a pity, because two different featured images come in very handy for bloggers.
